include/ocpp/v201/component_state_manager.hpp

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2020 - 2023 Pionix GmbH and Contributors to EVerest
+
+#pragma once
+
+#include <ocpp/v201/enums.hpp>
+#include "database_handler.hpp"
+
+namespace ocpp::v201 {
+
+struct FullConnectorStatus {
+    OperationalStatusEnum individual_operational_status;
+    bool faulted;
+    bool reserved;
```

Comment:
been there (I think `// cppcheck-suppress unusedStructMember` should suppress this)

Reply:
Thanks!